introduction.md

```diff
@@ -9,3 +9,9 @@ git Sleep
 git 2 HTML&CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?